cime_config/tests.py

```diff
@@ -41,6 +41,7 @@
         "time"  : "0:45:00",
         "tests" : (
             "ERS.f09_g16.IELMBC",
+            "ERS.f09_g16.IELMBC.elm-simple_decomp",
```

Comment:
Can this test go into `e3sm_land_exenoshare`? Because the test-mod is enabling multi-threading:
```
cat E3SM/components/elm/cime_config/testdefs/testmods_dirs/elm/simple_decomp/shell_commands 
./xmlchange NTHRDS=2
```
Default PEs on most machines are MPI-only (without OpenMP flags), and executables cannot be shared among threaded and non-threaded runs (non-BFB results otherwise).